apis/networking/v1beta1/apiproduct_types.go

```diff
@@ -87,6 +89,11 @@ func (a *APISelector) APINamespacedName() types.NamespacedName {
 	return types.NamespacedName{Namespace: a.Namespace, Name: name}
 }
 
+type InfraRateLimitSpec struct {
+	MaxValue int32 `json:"max_value"`
+	Period   int32 `json:"period_in_seconds"`
```

Comment:
```suggestion
	Period   int32 `json:"period"`
```

I'd call it just "period". We can specify that it should be set "in seconds" in the description of the field. Later in the future, if we want we can support humanized values as well, e.g. "1s", "3d", "1m", etc.